### DISCUSSION TITLE
How to move to an older release or a git branch

### DISCUSSION THREAD

**Discussion by Aggelos11** [ZerBea/hcxdumptool#445] [General]
How can I download and use an older version ?? How to handle the .tar.gz.asc file you provide ?

**Comment by ZerBea** [comment]
There are two options:

Via git as explained here;
https://git-scm.com/docs/git-checkout
After moving to the desired branch run
make clean
make
sudo make install

Via releases:
https://github.com/ZerBea/hcxdumptool/tags
Download tarball
Decompress tarball (e.g. tar -xf hcxdumptool-6.2.7.tar.gz)
cd hcxdumptool-6.2.7
make clean
make
sudo make install

In both cases do not forget to uninstall/remove the unwanted version.

Moved to discussions, because it is not an issue.
